### PR TITLE
Close gallery

### PR DESCRIPTION
**Pull request by davidpmccormick**
An idea for closing the image gallery. Putting it here so I don't forget/we've got something to push against.

close_gallery

### CODE REVIEW

**Comment by davidpmccormick** [comment]
@GarethOrmerod I can show you this next week if you have any design recommendations.

File: common/views/components/ImageGalleryV2/ImageGalleryV2.js [hunk 1]
```diff
@@ -131,6 +135,31 @@ class ImageGallery extends Component<Props, State> {
...
+                      visibility: isActive ? 'visible' : 'hidden',
+                      zIndex: 3,
+                    }}
+                  >
+                    <Control
+                      url={`#gallery-${id}`}
+                      type={`dark`}
+                      text={`close`}
+                      icon={`cross`}
+                      clickHandler={event => {
```
Comment (jamesgorrie) [inline common/views/components/ImageGalleryV2/ImageGalleryV2.js]: Tracking too?

File: common/views/components/ImageGalleryV2/ImageGalleryV2.js [hunk 2]
```diff
@@ -131,6 +135,37 @@ class ImageGallery extends Component<Props, State> {
...
+                  <div
+                    style={{
+                      position: 'sticky',
+                      top: '10px',
+                      paddingRight: '10px',
+                      paddingBottom: '10px',
+                      display: 'flex',
+                      justifyContent: 'flex-end',
+                      visibility: isActive ? 'visible' : 'hidden',
+                      zIndex: 3,
```
Comment (jamesgorrie) [inline common/views/components/ImageGalleryV2/ImageGalleryV2.js]: @wellcometrust/experience-devs 

When do we not want to be doing this?
I know it's a test, but we have classes for almost all of the above in `_root`?
There's a bigger question I think about how we use `_root` with `styled-components`?

Comment (davidpmccormick) [reply common/views/components/ImageGalleryV2/ImageGalleryV2.js]: Was wondering the same (about if/how to share styled-components and root-scope-classes).

Would be nice to have this using styled components, though.

Have dropped a `TODO` in there.

File: common/views/components/ImageGalleryV2/ImageGalleryV2.js [hunk 3]
```diff
@@ -136,34 +136,45 @@ class ImageGallery extends Component<Props, State> {
...
-                        trackEvent({
-                          category: `Control`,
-                          action: 'close ImageGallery',
-                          label: this.props.id,
-                        });
-
-                        setTimeout(() => {
-                          this.setState({ isActive: false });
-                        }, 0);
+                    <div
```
Comment (jamesgorrie) [inline common/views/components/ImageGalleryV2/ImageGalleryV2.js]: Not sure what aria attributes we might want to support here, if any, as the page would be rendered as a long list (I suppose) by screenreaders.

File: common/views/components/ImageGalleryV2/ImageGalleryV2.js (hunk 2 above)
Comment (jamesgorrie) [reply common/views/components/ImageGalleryV2/ImageGalleryV2.js]: Let's have a chat about this while chewing the code.

File: common/views/components/ImageGalleryV2/ImageGalleryV2.js [hunk 4]
```diff
@@ -136,34 +136,45 @@ class ImageGallery extends Component<Props, State> {
...
+                        text={`close`}
+                        icon={`cross`}
+                        clickHandler={event => {
+                          trackEvent({
+                            category: `Control`,
+                            action: 'close ImageGallery',
+                            label: this.props.id,
+                          });
+
+                          setTimeout(() => {
```
Comment (jamesgorrie) [inline common/views/components/ImageGalleryV2/ImageGalleryV2.js]: what's this little ditty about?

Comment (davidpmccormick) [reply common/views/components/ImageGalleryV2/ImageGalleryV2.js]: Good question – whatever it was about originally doesn't appear to be necessary any more. Will remove.

**Pull request merged**